wdl/ExpansionHunterDenovo.wdl

```diff
@@ -0,0 +1,347 @@
+## ExpansionHunter denovo (EHdn)
+##
+## This WDL implements workflows for
+## EHdn's two execution modes:
+## (i) case-control analysis;
+## (ii) outliner analysis.
+
+version 1.0
+
+import "Structs.wdl"
+
+struct FilenamePostfixes {
+  String locus
+  String motif
+  String profile
+  String merged_profile
+  Int profile_len
+}
+
+workflow EHdnSTRAnalysis {
+
+  input {
+    String analysis_type
+    String str_comparison_type
+    Array[File] case_reads_filenames
+    Array[File] case_indexes_filenames
+    Array[File] control_reads_filenames
+    Array[File] control_indexes_filenames
+    File reference_filename
```

Comment:
I would again avoid `filename` - usually we use `reference_fasta`. Also, does ehdn require this to be indexed? You may also need to include that (file extension `.fai`).